app/helpers/progress_tab_helper.rb

```diff
@@ -29,4 +29,13 @@ def create_badge_array(current_value)
 
     badges.reverse
   end
+
+  def numbers_to_relative_percentages(array)
```

Comment:
We should use https://github.com/simpledotorg/simple-server/blob/master/app/models/reports/percentage.rb#L3 to calculate the percentages, so it's consistent across the app.

Reply:
Great, thank you! Using the `percentage` method :)